ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_extrema_finding.py

```diff
@@ -344,3 +344,44 @@ def test_numpy_fmax(
         out=None,
         where=where,
     )
+
+# fmin
+@handle_frontend_test(
+    fn_tree="numpy.fmin",
+    dtype_and_inputs=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
```

Comment:
where relevant (ie, functions should accept integer etc parameters and not only float), please use `numeric` instead

Reply:
replaced this